docs/react/api/hooks/useCall.md

```diff
@@ -0,0 +1,397 @@
+    data: '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
+    maxFeePerGas: parseGwei('20'),
+    maxPriorityFeePerGas: parseGwei('2'), // [!code focus]
+    to: '0x70997970c51812dc3a010c7d01b50e0d17dc79c8',
+  })
+}
+```
+<<< @/snippets/react/config.ts[config.ts]
+:::
+
+### nonce
+
+`number | undefined`
+
+Unique number identifying this transaction.
+
+::: code-group
+```tsx [index.tsx]
+import { useCall } from 'wagmi'
+
+function App() {
+  const result = useCall({
+    account: '0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266',
+    data: '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
+    nonce: 420, // [!code focus]
+    to: '0x70997970c51812dc3a010c7d01b50e0d17dc79c8',
+  })
+}
+```
+<<< @/snippets/react/config.ts[config.ts]
+:::
+
+### value
+
+`bigint | undefined`
+
+Value (in wei) sent with this transaction.
+
+::: code-group
+```tsx [index.tsx]
+import { useCall } from 'wagmi'
+
+function App() {
+  const result = useCall({
+    account: '0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266',
+    data: '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
+    to: '0x70997970c51812dc3a010c7d01b50e0d17dc79c8',
+    value: parseEther('1'), // [!code focus]
+  })
+}
+```
+<<< @/snippets/react/config.ts[config.ts]
+:::
+
+### blockNumber
+
+`number | undefined`
+
+The block number to perform the call against.
+
+::: code-group
+```tsx [index.tsx]
+import { useCall } from 'wagmi'
+
+function App() {
+  const result = useCall({
+    blockNumber: 15121123n, // [!code focus]
+    account: '0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266',
+    data: '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
+    to: '0x70997970c51812dc3a010c7d01b50e0d17dc79c8',
+  })
+}
+```
+<<< @/snippets/react/config.ts[config.ts]
+:::
+
+### blockTag
+
+`'latest' | 'earliest' | 'pending' | 'safe' | 'finalized' | undefined`
+
+The block tag to perform the call against.
+
+::: code-group
+```tsx [index.tsx]
+import { useCall } from 'wagmi'
+
+function App() {
+  const result = useCall({
+    blockTag: 'safe', // [!code focus]
+    account: '0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266',
+    data: '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
+    to: '0x70997970c51812dc3a010c7d01b50e0d17dc79c8',
+  })
+}
+```
+<<< @/snippets/react/config.ts[config.ts]
+:::
+
+### chainId
+
+`config['chains'][number]['id'] | undefined`
+
+The block tag to perform the call against.
+
+::: code-group
+```tsx [index.tsx]
+import { useCall } from 'wagmi'
+import { mainnet } from '@wagmi/core/chains'
+
+function App() {
+  const result = useCall({
+    chainId: mainnet.id, // [!code focus]
+    account: '0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266',
+    data: '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
+    to: '0x70997970c51812dc3a010c7d01b50e0d17dc79c8',
+  })
+}
+```
+<<< @/snippets/react/config.ts[config.ts]
+:::
+
+### config
+
+`Config | undefined`
+
+[`Config`](/react/api/createConfig#config) to use instead of retrieving from the from nearest [`WagmiProvider`](/react/WagmiProvider).
```

Comment:
```suggestion
[`Config`](/react/api/createConfig#config) to use instead of retrieving from the from nearest [`WagmiProvider`](/react/api/WagmiProvider).
```